pkg/kv/kvserver/scheduler.go

```diff
@@ -344,13 +348,19 @@ func (s *raftScheduler) worker(ctx context.Context) {
 func (s *raftScheduler) enqueue1Locked(
 	addFlags raftScheduleFlags, id roachpb.RangeID, now int64,
 ) int {
+	// TODO(pavelkalinnikov): Pass ticks increment to avoid branching below.
+	ticked := addFlags&stateRaftTick != 0
+
 	prevState := s.mu.state[id]
-	if prevState.flags&addFlags == addFlags {
+	if prevState.flags&addFlags == addFlags && !ticked {
 		return 0
 	}
 	var queued int
 	newState := prevState
 	newState.flags = newState.flags | addFlags
+	if ticked {
+		newState.ticks++ // TODO(pavelkalinnikov): Check overflows, or have a limit.
```

Comment:
I wouldn't worry about overflow. If we're ticking at 200 ms, it'll overflow in 10 billion years.

Reply:
Good point. It's 13.6 years though (if I'm not mistaken that max `int` is 2^31-1). It still seems enough, but a bit on the edge (esp. if tick time is tuned down to order of milliseconds).

Reply:
`int` defaults to 64-bit on 64-bit architectures. I think that's the only one we really care about, not sure if we even support 32-bit.